src/validations/auth.validation.js

```diff
@@ -40,15 +40,11 @@ module.exports = {
         'string.pattern.base': `Contraseña debería tener entre 8 y 30 carácteres además de contener letras y números`
       }),
       confirmPassword: joi
-        .string()
+        .any()
         .valid(joi.ref('password'))
         .required()
-        .pattern(new RegExp("^[a-zA-z0-9]{8,30}$"))
         .messages({
-          'any.required': `Confirmar contraseña es requerida`,
-          'string.base': `Confirmar contraseña debe ser un texto válido`,
-          'string.empty': `Confirmar contraseña no debe estar vacía`,
-          'string.pattern.base': `Confirmar contraseña debería tener entre 8 y 30 carácteres además de contener letras y números`
+          'any.only': `Contraseñas no coinciden`,
```

Comment:
Porque el cambio? Antes los mensajes estaban mejor , mas específicos de cual es el problema.

Reply:
Because the validations are in the password field and confirm password must match with password

Reply:
Si entiendo, no me había fijado que era para el cambio de contraseña, thanks